### ISSUE TITLE
SC2034, SC1068 false positives with tr + equal sign

### ISSUE THREAD

**Issue by Gandalf-** [koalaman/shellcheck#1663]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
SC2034, SC1068
- My shellcheck version (`shellcheck --version` or "online"): 
```
ShellCheck - shell script analysis tool
version: 0.7.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

data="
icmp_seq=1
icmp_seq=2
icmp_seq=3
icmp_seq=4
icmp_seq=5
icmp_seq=6
icmp_seq=7
icmp_seq=8
icmp_seq=9
"
# the here string isn't important, could have been cat, redirection, etc

tr = ' ' <<< "$data"
```

#### Here's what shellcheck currently says:
```
I leaf@elm /m/z/G/c/shell> shellcheck example.sh

In example.sh line 15:
tr = ' ' <<< "$data"
^-^ SC2034: tr appears unused. Verify use (or export if used externally).
   ^-- SC1068: Don't put spaces around the = in assignments (or quote to make it literal).

For more information:
  https://www.shellcheck.net/wiki/SC1068 -- Don't put spaces around the = in ...
  https://www.shellcheck.net/wiki/SC2034 -- tr appears unused. Verify use (or...
```

#### Here's what I wanted or expected to see:
Potentially nothing. This is a bit of weird case, but we may be able to add a couple exceptions to these checks for commands like `tr` or `grep` that take character ranges.

This seems like a low priority bug 🙂

**Comment by Nightfirecat** [comment]
SC1068 seems to be accurate, specifically the `(or quote to make it literal)` part of it. SC2034 is probably a minor bug/ambiguity because of what appears at a glance to be an assignment statement.
